packages/overlay/src/overlay-trigger.ts

```diff
@@ -39,12 +31,13 @@ let overlayRoot: OverlayRoot;
  * @slot click-content - The content that will be displayed on click
  */
 export class OverlayTrigger extends LitElement {
+    private clickOverlay?: Overlay;
+    private hoverOverlay?: Overlay;
```

Comment:
The only method we leverage on these `Overlay`s is `close`...do we see the possibility of expanding to others later? Should we just store the close method instead of the whole overlay?

Reply:
I have updated the code as per this suggestion. I think it is good to hide the implementation details of the overlay until we think of a reason why we would need to access the overlay object.